examples/validate_relative_binding.py

```diff
@@ -45,6 +45,26 @@
 from md import builders, minimizer
 from rdkit.Chem import rdFMCS
 
+
+def cache_wrapper(cache_path: str, f):
```

Comment:
Feels like we might want to put this somewhere else so we can use this sort of logic in other scripts.